internal/vendor.json

```diff
@@ -1,269 +1,320 @@
-			"VersionTime": "2015-06-30T10:09:15-06:00"
+			"canonical": "github.com/rackspace/gophercloud/rackspace/networking/v2/subnets",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/rackspace/networking/v2/subnets",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
 		},
 		{
-			"Vendor": "github.com/rackspace/gophercloud/rackspace/objectstorage/v1/objects",
-			"Local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/rackspace/objectstorage/v1/objects",
-			"Version": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
-			"VersionTime": "2015-06-30T10:09:15-06:00"
+			"canonical": "github.com/rackspace/gophercloud/rackspace/objectstorage/v1/containers",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/rackspace/objectstorage/v1/containers",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
 		},
 		{
-			"Vendor": "github.com/rackspace/gophercloud/testhelper",
-			"Local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/testhelper",
-			"Version": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
-			"VersionTime": "2015-06-30T10:09:15-06:00"
+			"canonical": "github.com/rackspace/gophercloud/rackspace/objectstorage/v1/objects",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/rackspace/objectstorage/v1/objects",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
 		},
 		{
-			"Vendor": "github.com/rackspace/gophercloud/testhelper/client",
-			"Local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/testhelper/client",
-			"Version": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
-			"VersionTime": "2015-06-30T10:09:15-06:00"
+			"canonical": "github.com/rackspace/gophercloud/testhelper",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/testhelper",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
 		},
 		{
-			"Vendor": "gopkg.in/ini.v1",
-			"Local": "github.com/jrperritt/rack/internal/gopkg.in/ini.v1",
-			"Version": "177219109c97e7920c933e21c9b25f874357b237",
-			"VersionTime": "2015-03-19T00:52:36-04:00"
+			"canonical": "github.com/rackspace/gophercloud/testhelper/client",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/testhelper/client",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
+		},
+		{
+			"canonical": "gopkg.in/ini.v1",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/gopkg.in/ini.v1",
+			"revision": "177219109c97e7920c933e21c9b25f874357b237",
+			"revisionTime": "2015-03-19T00:52:36-04:00"
```

Comment:
Hmm, I hadn't noticed that. It looks like this new `vendor.json` has the correct format though (according to https://github.com/kardianos/vendor-spec#example)